packages/components/tests/dummy/app/templates/components/table.hbs

```diff
@@ -2,6 +2,13 @@
 <h2 class="dummy-h2">
   Table
 </h2>
+<section>
+  <div class="dummy-banner dummy-banner--info">
+    <p class="dummy-paragraph">⚠️
+      <strong>Note</strong>: this component has some features that are still in flux. If you find that your particular
```

Comment:
```suggestion
      <strong>Note</strong>: this component has some features that are still in flux (more specifically: alignment and width of cells, and automatic sorting of columns). If you find that your particular
```

Reply:
@didoo the automatic sorting of columns works, what scenario specifically are you thinking about?

Reply:
Based on what we were discussing in our meeting and in slack just now, I think it's likely the API for sorting will change slightly away from an array of keys to having whether or not something is sortable be part of the `columns` object.

Reply:
Ahh, I see what is intended now. Let me fix, one min.